docs/update_benchmark_tables.py

```diff
@@ -132,19 +143,26 @@ def benchmark_result_to_task_type_row(
 
     scores = []
     task_type_names = []
+    n_datasets = []
     for t, ts in tasktype2tasks.items():
         task_results = [r for r in result.task_results if r.task_name in ts]
         _scores = np.array([get_main_score(t, langs) for t in task_results])  # type: ignore
         score = np.mean(_scores)
         scores.append(score)
-        task_type_names.append(t.capitalize() + " (#Tasks=" + f"{len(ts)})")
+        task_type_names.append(t.capitalize())
+        n_datasets.append(len(ts))
 
-    mdl_name = create_mdl_name(result.meta)
+    mdl_name = create_mdl_name_w_reference(result.meta)
     df = pd.DataFrame([scores], columns=task_type_names, index=[mdl_name])
     df["Average Score"] = result.get_main_score() * 100
     df["Open Source"] = open_source_to_string(result.meta.open_source)
     df["Embedding Size"] = result.meta.embedding_size
-    df["Speed (CPU)"] = get_speed_results(result.meta)
+    df["WPS (CPU)"] = get_speed_results(result.meta)
+
+    # create row for number of datasets
+    df2 = pd.DataFrame([n_datasets], columns=task_type_names, index=["N. Datasets"])
```

Comment:
It's fine for now but I really don't like that you called this `df2` and then have to explain in a comment, why you did it.
I would probably also do it before adding the additional columns to the dataframe, so something like:

```python
df = pd.DataFrame([scores], columns=task_type_names, index=[mdl_name])
df.loc["N. Datasets"] = n_datasets
df["Average Score"] = result.get_main_score() * 100
...
```

Reply:
agree fixed in new commit